frontend/tests/utils/downloadUtils.ts

```diff
@@ -424,3 +447,13 @@ export async function downloadGeneFile(
     zip.extractAllTo(dirPath);
   }
 }
+
+// Gets the date in mmddyy format
```

Comment:
maybe worth adding a comment on why we copied this from product code?

E.g. or something better,
```suggestion
/**
 * (ashin-czi): Gets the date in mmddyy format
 * Copied from `frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveExport/csvUtils.ts`
 * since Playwright doesn't like importing from that file due to its dependency on `d3`
 */
```